### ISSUE TITLE
Donate widget a11y

### ISSUE THREAD

**Issue by LouiaR** [comicrelief/component-library#270]
Issues: 
• This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.59:1. Recommendation:  change background to #6775a2.
• This fieldset element does not have a name available to an accessibility API. Valid names are: legend element, aria-label undefined, aria-labelledby undefined.
 • Fieldset does not contain a legend element. All fieldsets should contain a legend element that describes a description of the field group.
